schema/rdl_schema_0.1.json

```diff
@@ -125,6 +142,10 @@
 									"type": "string",
 									"description": "The web address the dataset version can be downloaded from.",
 									"format":"uri"
+								},
+								"period": {
+									"type": "string",
+									"description": "The period of time covered by this dataset version given as a year or range of years."
```

Comment:
We should consider a more structured model for this field, e.g. a `period` object with `.start` and `.end` properties. That would make it easier for users and tools to filter for datasets covering a specific year.

Reply:
Continued in https://github.com/GFDRR/rdl-standard/issues/54#issuecomment-1553959738